sdk/identity/identity-broker/samples-dev/index.ts

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * @summary Demonstrates usage of @azure/identity-broker and @azure/identity
+ * packages for WAM MSA support.
+ */
+
+import {
+  InteractiveBrowserCredential,
+  InteractiveBrowserCredentialNodeOptions,
+  useIdentityPlugin,
+} from "@azure/identity";
+import { createNativeBrokerPlugin } from "@azure/identity-broker";
+import dotenv from "dotenv";
+import { BrowserWindow } from "electron";
+const win = new BrowserWindow();
+
+// Load the plugin
+useIdentityPlugin(createNativeBrokerPlugin());
+
+// Load the environment
+dotenv.config();
+
+async function main() {
+  const options: InteractiveBrowserCredentialNodeOptions = {
+    clientId: process.env.AZURE_CLIENT_ID || "client",
+    authorityHost: process.env.AZURE_AUTHORITY_HOST,
+    tenantId: process.env.AZURE_TENANT_ID,
+    brokerOptions: {
+      enableMSAPassthrough: true,
+      parentWindowHandle: Uint8Array.from(win.getNativeWindowHandle()),
+    },
+  };
+  const credential = new InteractiveBrowserCredential(options);
+
+  // This is the scope we will use to get a token from the AAD token endpoint.
```

Comment:
@scottaddie do we need to start calling this Entra in comments?

Reply:
done